src/system/WatchableEventManagerSelect.cpp

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
Derived from `WatchableSocketSelect.cpp`; see https://gist.github.com/kpschoedel/e97522d8b251a7a30d1735ecd765d56d for diff